unic-langid-likelysubtags/src/lib.rs

```diff
@@ -1,12 +1,24 @@
-mod tables;
+pub mod tables;
 
+use tinystr::{TinyStr4, TinyStr8};
 use unic_langid::LanguageIdentifier;
 
+fn get_lang_from_parts(
```

Comment:
hmm, why? It's an internal function called within add_likely/remove_likely on known (generated) data.
If I make this function unsafe, then all callsites will have to wrap it. It feels less ergonomic, and since the function is not public, doesn't affect the public API.

Am I missing something?

Reply:
Typically you still mark things as unsafe internally up to the point where you can ensure the invariants are upheld

Put another way, every use of `unsafe` could/should have a comment explaining why the invariants are being upheld. If you don't have a way to do that (e.g. here, where the invariants are being upheld elsewhere) you probably need to expand the usage

Reply:
thanks! filed as a new issue.